notification/test/unit/hmac-verification.spec.js

```diff
@@ -0,0 +1,26 @@
+const { cloneDeep } = require('lodash')
+const { expect } = require('chai')
+const { hasValidHMACSignature } = require('../../src/utils/hmacValidator')
+const notificationRequest = require('../resources/notification.json')
+const notification = notificationRequest.notificationItems
+
+describe('verify hmac signatures', () => {
```

Comment:
These tests are failing on my local computer. You should set `process.env.ADYEN_SECRET_HMAC_KEY` in those tests directly with the valid value.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/273/commits/e84833839a96382028572dd8535ad5ee05a15582